pkg/storage/mysql/mysql_test.go

```diff
@@ -27,30 +25,6 @@ func TestMySQLDatastore(t *testing.T) {
 	test.RunAllTests(t, ds)
 }
 
-func TestMigrate(t *testing.T) {
```

Comment:
FYI i moved these tests so that there is no cycle in the import loop

Reply:
Where do we move these tests to?  I cannot find it in this review

Reply:
I forgot to commit that file 🤣  added! The test is `TestMigrateCommandRollbacks`